public-docs/devs-update-custom-plugin.md

```diff
@@ -0,0 +1,87 @@
+---
+id: devs-update-custom-plugin
```

Comment:
This is great work! I'd be curious to have @zenweasel or someone on the Sports Direct team and some of our community review this after working to update a plugin.

Reply:
Good idea. I'm certain there are more changes to add here, but we'll need to sift through the final change log to complete this at the very end.